includes/class-wordpress-template-integration.php

```diff
@@ -250,6 +250,11 @@ public function display_view_in_content( $content = '' ) {
 		$is_tax = is_tax();
 
 		if ( ! $is_tax ) {
+            // This filter is added because of an issue with remove_filter. 
+            // See: https://github.com/Strategy11/business-directory-plugin/pull/400
+            add_filter( 'the_content', fn( $content ) => $content, 1000 );
```

Comment:
Looks like this format requires php 7.4, but we are still supporting previous versions.

Reply:
Right, I thought we support 7.4, but we support 7.0. I'll keep that in mind. Thanks @stephywells!